client/Components/lodging.jsx

```diff
@@ -1,42 +1,44 @@
 import React from 'react';
 
+const types = ['hc-awesome-tent', 'hc-awesome-house', 'hc-awesome-tent', 'hc-awesome-tent', 'hc-awesome-cabin', 'hc-awesome-rv']
```

Comment:
Again another thing I would love to learn!

Reply:
this is just an array that will reference a certain glyph that has been aliased in my css file